vms/platformvm/warp/payload/README.md

```diff
@@ -0,0 +1,44 @@
+# Payload
+
+An Avalanche Unsigned Warp Message already includes a `networkID`, `sourceChainID`, and `payload` field. The `payload` field is parsed into one of the types included in this package to be further handled by the VM.
+
+## AddressedPayload
+
+AddressedPayload:
+```
++---------------------+--------+----------------------------------+
+|             codecID : uint16 |                          2 bytes |
++---------------------+--------+----------------------------------+
+|              typeID : uint32 |                          4 bytes |
++---------------------+--------+----------------------------------+
+|       sourceAddress : []byte |                 4 + len(address) |
++---------------------+--------+----------------------------------+
+|             payload : []byte |                 4 + len(payload) |
++---------------------+--------+----------------------------------+
+                               | 14 + len(payload) + len(address) |
+                               +----------------------------------+
+```
+
+- `codecID` is the codec version used to serialize the payload and is hardcoded to `0x0000`
+- `typeID` is the payload type identifier and is `0x00000000` for `AddressedPayload`
+- `sourceAddress` is the address that called `sendWarpPrecompile` on the source chain
+- `payload` is an arbitrary byte array payload
+
+## BlockHashPayload
+
+BlockHashPayload:
```

Comment:
Is there a reason this is `BlockHashPayload` rather than a `Hash` payload? Seems unnecessarily specific.

Reply:
This is meant to be specific to the BlockHash as opposed to an arbitrary hash. If we changed to an arbitrary hash, the caller would need to have a different way to differentiate between hash types 🤷 .

I don't think there's a need for an arbitrary hash payload at this time, so lmk if you'd prefer I change it anyways.

Reply:
Going to leave this as `BlockHash` after removing the `Payload` suffix.